pages/windows/dir.md

```diff
@@ -17,3 +17,7 @@
 - Show the contents of a given directory, including hidden ones:
 
 `dir {{path/to/directory}} /A`
+
+- Also show the contents of all subdirectories:
+
+`dir {{path/to/directory}} /A`
```

Comment:
This example looks the same as the last?

Reply:
yikes. that happens when you are tired and use the web ui, sorry...
sadly, the edit function in the files changed tab requires push access to the repo, so I cannot edit it
Will try if I can get it to work with the full git pipeline but it has to wait, sorry.